src/features/apiDelegation/offered/ChooseOrgPage/ChooseOrgPage.tsx

```diff
@@ -293,7 +299,7 @@ export const ChooseOrgPage = () => {
                   <h4 className={classes.chosenOrgsHeader}>
                     {t('api_delegation.businesses_going_to_get_access')}
                   </h4>
-                  <div className={classes.actionBarScrollContainer}>{chosenItems}</div>
```

Comment:
It's still scrollable, it is removed because it broke spacing, it's bad for our phone users and because delegating to many organizations isn't that common